src/display/views/threads/components/__tests__/CheckboxTable.spec.js

```diff
@@ -1,7 +1,7 @@
 // #region imports
 import React from 'react';
-import { shallow, mount } from 'enzyme';
 import CheckboxTable from '../CheckboxTable';
+import { shallowWithState, mountWithState } from '../../../../../../config/tests/helpers.unit';
```

Comment:
I wasn't too happy with how invasive `connect`ing the CheckboxTable to the store was to the tests. However, the alternative seems to be making all five consumers of ThreadTable (who are already connected to the store) pluck the same theme state and pass it down. It seemed better for CheckboxTable to manage its own dependencies, but ehhh. Let me know what you think!

Reply:
I take your point and I like the thoughtfulness on the choice! Looking at the `ThreadTable` consumers, though, they're all passing `{...this.props}` as a prop to `ThreadTable` anyway. So I think I prefer just having another forwarded key in the various `mapStateToProps` methods rather than pushing `connect` further down the component stack; I prefer keeping that as high as possible with more "dummy" display components underneath. (especially since, as you say, it helps keep the tests simple.)

Reply:
👍 Switched out last commit for that approach